### DISCUSSION TITLE
ring_log

### DISCUSSION THREAD

**Discussion by zV3KT0R** [yanet-platform/yanet#202] [Q&A] [answered]
who does dequeueing for ring_log

https://github.com/yanet-platform/yanet/blob/87748d7bf95f44ba21725197b0ef18e196310741/dataplane/worker.cpp#L5582

**Comment by GeorgyKirichenko** [comment, marked as answer]
There is secondary application called logger which reads from the queue

**Comment by zV3KT0R** [comment]
OK, thanks
